tso-server/src/main/java/org/apache/omid/tso/ReplyProcessorImpl.java

```diff
@@ -67,7 +69,8 @@
     private final Meter timestampMeter;
 
     @Inject
-    ReplyProcessorImpl(MetricsRegistry metrics, Panicker panicker, ObjectPool<Batch> batchPool) {
+    ReplyProcessorImpl(@Named("PersistenceStrategy") WaitStrategy strategy,
```

Comment:
Typo in named. Should be the same name that is in the DisruptorModule.java for the reply processor:

```
@Named("ReplyStrategy")
```